### ISSUE TITLE
Platform not found: sensor.airthings_wave

### ISSUE THREAD

**Issue by johntdyer** [custom-components/sensor.airthings_wave#16]
Home Assistant 0.96.2

Airthings Wave: aeabf69

```
root@ubuntu:/usr/share/hassio/homeassistant/custom_components/airthings_wave# hass --script check_config -c  /usr/share/hassio/homeassistant/
Testing configuration at /usr/share/hassio/homeassistant/
Failed config
  General Errors:
    - Platform not found: sensor.airthings_wave

Successful config (partial)
```

This is after numerous restarts.   Any idea what I am doing wrong ?

**Comment by MartyTremblay** [comment]
Yes, I'm noticing the same thing on my side. It may be related to a bluetooth issue that some folks are reporting since release 95. When I revert to release 94, it is working for me. Are using hassos?

**Comment by johntdyer** [comment]
@MartyTremblay  - yes I am using hassio 0.96.3.  Do you have a link to the BLE issues you referred too ?

**Comment by MartyTremblay** [comment]
This is what I had stumbled on. https://github.com/home-assistant/home-assistant/issues/24605

**Comment by debsahu** [comment]
I'm on 0.96.5 and it is affected by aforementioned bluetooth issue. It reads for the first reading and fails for subsequent readings

```
2019-07-29 08:16:34 ERROR (Basement Airthigs Wave) [pygatt.backends.gatttool.gatttool] Timed out connecting to XX:XX:XX:XX:XX after 30 seconds.
2019-07-29 08:16:34 ERROR (Basement Airthigs Wave) [custom_components.airthings_wave.sensor] Exception: Timed out connecting to XX:XX:XX:XX:XX after 30 seconds. 
```

![image](https://user-images.githubusercontent.com/24575521/62067438-bf8e5b80-b201-11e9-9281-eea67a234a53.png)

If I restart HA, it reads the first reading and I get the timeout error from gatttool subsequently.

**Comment by MartyTremblay** [comment]
Seems to have been resolved since 0.97 +. I've been running it on 0.98.x for more than a week after a reinstall of the component with no issues. Can someone confirm before I close this ticket?

**Comment by fleshinchair** [comment]
I meant to reply earlier but yes it is now working for me too! Great work!
 Thankyou for fixing this, it's a great component and makes the device a lot more useful
